egs/ami/s5/local/nnet3/run_lstm.sh

```diff
@@ -1,4 +1,3 @@
-#!/bin/bash
 
 # Copyright 2015  Johns Hopkins University (Author: Daniel Povey).
 #           2015  Vijayaditya Peddinti
```

Comment:
it would probably make sense to mention that this is also called from other scripts in local/nnet3/, such as run_blstm.sh.